client/modules/IDE/components/EditorAccessibility.js

```diff
@@ -7,18 +7,19 @@ class EditorAccessibility extends React.Component {
   render() {
     let messages = [];
     if (this.props.lintMessages.length > 0) {
-      for (let i = 0; i < this.props.lintMessages.length; i++) {
+      this.props.lintMessages.forEach((lintMessage, i) => {
         messages.push(
-          <li>
-            {this.props.lintMessages[i].severity} in line
-            {this.props.lintMessages[i].line} :
-            {this.props.lintMessages[i].message}
+          <li key={i}>
+            {lintMessage.severity} in line
+            {lintMessage.line} :
+            {lintMessage.message}
           </li>
         );
-      }
+      });
     } else {
       messages.push(
-        <p tabIndex="0"> There are no lint messages </p>
+        // no array index so let's just call it 0
+        <p tabIndex="0" key={0}> There are no lint messages </p>
```

Comment:
Is there a reason this is a <p> and not a <li>, if it's going to go inside a <ul>?